jenkins/Jenkinsfile

```diff
@@ -17,16 +17,16 @@ pipeline {
             steps {
                 sh '''#!/usr/bin/env bash
                 set -e
-                cp {{cookiecutter.project_slug}}/setup_miniconda.sh .
+                cp {{ project_slug }}/setup_miniconda.sh .
                 bash setup_miniconda.sh -p ${WORKSPACE}
                 '''
                 sh '''#!/usr/bin/env bash
                 set -e
                 source $WORKSPACE/miniconda/etc/profile.d/conda.sh
-                conda create -n cookiecutter
-                conda activate cookiecutter
-                conda install -c conda-forge cookiecutter
-                cookiecutter --no-input .
+                conda create -n copier
```

Comment:
@regDaniel make launch jenkins to test this

Reply:
done